reference/user_defined_functions.rst

```diff
@@ -36,6 +107,8 @@ This example informs the database that the github_events table should be distrib
 
 	SELECT master_create_distributed_table('github_events', 'repo_id', 'hash');
 
+.. note::
```

Comment:
Could we move this note to the top like the one we have for [rebalancer functions](https://citus-doc-preview-pr-190.herokuapp.com/reference/user_defined_functions.html#rebalance-table-shards)?

This way users can read the note before they run the function. It shouldn't be the case that they learn this after they've already used this function to distribute their tables.